src/EditorFeatures/Core/Implementation/Classification/IEditorClassificationService.cs

```diff
@@ -1,111 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Classification;
-using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Text;
-
-namespace Microsoft.CodeAnalysis.Editor
-{
-    [Obsolete("Use Microsoft.CodeAnalysis.Classification.IClassificationService instead")]
```

Comment:
I am not planing to check this in to preview 2.

Reply:
since it got blocked, I couldn't check this in and now it is too late. due to all the change came in, I need to retest everything and work with FSharp team to verify change again. forcing the PR to move to 16.1